text/0061-promote-ubi-stack.md

```diff
@@ -0,0 +1,56 @@
+# Promote ubi stack to paketo-buildpacks
+
+## Summary
+
+Addition of ubi stacks was discussed and approved in
+[https://github.com/paketo-buildpacks/rfcs/blob/main/text/0056-ubi-based-stacks.md])https://github.com/paketo-buildpacks/rfcs/blob/main/text/0056-ubi-based-stacks.md).
+
+Initially repositories were added to to paketo-community to reflect that they were under early development. 
+
+The ubi stack has been being built and published since August 2023. Except where is implements
+advanced features like multi-arch it uses the common automation, and the team is working to integrate
+the advanced featurs back into the common versions of the automation flows.  
```

Comment:
```suggestion
The ubi stack has been being built and published since August 2023. Except where it implements
advanced features like multi-arch it uses the common automation, and the team is working to integrate
the advanced features back into the common versions of the automation flows.  
```